patterns/2-structured/praise-participants.md

```diff
@@ -4,59 +4,60 @@ Praise Participants
 
 ## Patlet
 
-After an inner source contribution, it's important to thank the contributor for their time and effort.
-This pattern gives guidance that not only effectively acknowledges the contribution but also engenders further engagement from the contributor and others.
+When you receive an InnerSource contribution, it's important to thank the contributor for their time and effort.
+Extending your gratiutude not only effectively acknowledges the contribution but also engenders further engagement from the contributor and others.
+Praising contributors' positive contributions to your InnerSource project motivates those contributors (and their managers) to continue investing in the effort.
 
 ## Problem
 
-How can we properly express our gratitude to a contributor for their inner source contribution to a project?
-It can be easy to forget to do so or not know the words or medium to use for adequate effect and sincerity.
-Praise and thanks are easy, affordable ways to keep contributors and their managers motivated and excited to continue.
```

Comment:
The bit about "their mangers" in here seems like an interesting bit of info. Can we somehow keep this?

Reply:
I see what you mean. Let me attempt a re-write and we'll see what you think.

Reply:
Okay! New version is in place. What do you think?